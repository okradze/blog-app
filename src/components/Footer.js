import React from 'react';
import { Link } from 'react-router-dom';

export const Footer = () => (
	<footer className="footer">
		<div className="container footer__content">
			<nav className="footer__nav">
				<Link className="footer__link" to="/dashboard">
					Dashboard
				</Link>
				<Link className="footer__link" to="/create">
					Create
				</Link>
				<Link className="footer__link" to="/read">
					Blogs
				</Link>
			</nav>
			<p className="footer__copy">Copyright &copy; 2019 Mirian Okradze</p>
		</div>
	</footer>
);

export default Footer;
