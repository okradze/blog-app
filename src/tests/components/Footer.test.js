import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../components/Footer';

test('should render component correctly', () => {
	const wrapper = shallow(<Footer />);
	expect(wrapper).toMatchSnapshot();
});
