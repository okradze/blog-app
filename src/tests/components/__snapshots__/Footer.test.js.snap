// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component correctly 1`] = `
<footer
  className="footer"
>
  <div
    className="container footer__content"
  >
    <nav
      className="footer__nav"
    >
      <Link
        className="footer__link"
        replace={false}
        to="/dashboard"
      >
        Dashboard
      </Link>
      <Link
        className="footer__link"
        replace={false}
        to="/create"
      >
        Create
      </Link>
      <Link
        className="footer__link"
        replace={false}
        to="/read"
      >
        Blogs
      </Link>
    </nav>
    <p
      className="footer__copy"
    >
      Copyright Â© 2019 Mirian Okradze
    </p>
  </div>
</footer>
`;
